biopet-core/src/main/scala/nl/lumc/sasc/biopet/core/MultiSampleQScript.scala

```diff
@@ -141,11 +141,17 @@ trait MultiSampleQScript extends SummaryQScript { qscript: QScript =>
 
     /** returns a set with library names or throws error when not found */
     protected def libIds: Set[String] = {
-      val ids = ConfigUtils.getMapFromPath(globalConfig.map, List("samples", sampleId, "libraries")).getOrElse(Map()).keySet
-      if (ids.isEmpty) {
-        throw new IllegalStateException(s"No libraries found in config for sample $sampleId")
+      val ids: Set[String] = try {
+        ConfigUtils.getMapFromPath(globalConfig.map, List("samples", sampleId, "libraries")).getOrElse(Map()).keySet
+      } catch {
+        case e: IllegalStateException if e.getMessage == "Value is not a map: library" =>
+          Logging.addError("libraries is not a map")
```

Comment:
This is a somewhat nebulous error message if one is not familiar with the internal structure we use here. Perhaps simply state the contents are wrongly formatted?